tensorflow_datasets/core/dataset_utils.py

```diff
@@ -177,8 +193,15 @@ def _graph_dataset_iterator(ds_iter, graph=None):
         break
 
 
+@typing.overload
```

Comment:
I think you need to add the `# pylint: disable=` too.

Reply:
I don't think that is needed. Pylint is passing 100%.